CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
 # Contributing to Poetry
 
-First off, thank for taking the time to contribute!
+First off, thanks for taking the time to contribute!
 
-The following is a set of guidelines for contributing to Poetry on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to Poetry on GitHub. These are mostly guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request.
 
-#### Table of Contents
+#### Table of contents
```

Comment:
Remain [title case](https://titlecase.com)?

Reply:
I removed title case to be consistent with all the other headings which are not

Reply:
Pardon me that I ignored it's English here.